docs/upgrading-solidus/v3.3.mdx

```diff
@@ -11,7 +11,7 @@ import MinimalRequirements from '@site/src/theme/MinimalRequirements';
 
 New year, new Solidus release! This time, we acknowledge our [new policy around Ruby & Rails support](https://solidus.io/release_policy/). It also comes with new features, bug fixes, as well as some deprecations that will be removed in the next major release. Take the time to upgrade. We've committed with the community to release more often, so keeping up to date will help you when Solidus v4.0 is out, which will happen sooner than later.
 
-Before getting hands-on, please review our generic [upgrade guides](/getting-started/upgrading-solidus.mdx). You can also check the complete [Changelog](https://github.com/solidusio/solidus/blob/v3.3/CHANGELOG.md) in our repository.
+Before getting hands-on, please review our generic [upgrade guides](upgrading-solidus/index.mdx). You can also check the complete [Changelog](https://github.com/solidusio/solidus/blob/v3.3/CHANGELOG.md) in our repository.
```

Comment:
I noticed some links are absolute and others (like this) are relative, not sure is wrong or intentional, but wanted to point that out in case it's a typo

Reply:
It works anyway but better to be consistent. Thanks for pointing it out!